variables.tf

```diff
@@ -198,11 +198,11 @@ variable "enable_platform_metrics" {
 }
 
 variable "cloud_monitoring_service_endpoints" {
-  description = "The type of the service endpoint that will be set for the IBM Cloud Logs instance. Allowed values: public-and-private"
+  description = "The type of the service endpoint that will be set for the IBM Cloud Monitoring instance. Allowed values: public-and-private"
   type        = string
   default     = "public-and-private"
   validation {
-    condition     = contains(["public-and-private"], var.cloud_monitoring_service_endpoints)
+    condition     = contains(["public", "private", "public-and-private"], var.cloud_monitoring_service_endpoints)
```

Comment:
Yes, all of these are supported. I have checked these using the IBM Cloud CLI, `public` and `private` endpoints were getting created.

Reply:
But I dont think `public` only or `private` only works. Did you verify that?
I think regardless of what value you enter, it will always create both pubic and private, which would mean we need to remove the ability for user to specify `public` or `private` from this input.
You cannot specify this in the UI when provisioning, so this tells me these are probably not supported values (even if terraform allows you to enter them - it will always create both)

Reply:
Yes, even if the instance were showing as `private` when I fetched the cloud monitoring instance details, the created endpoints were `public`. So, the only supported input is `public-and-private`.